synapse/lib/stormtypes.py

```diff
@@ -7381,28 +7388,24 @@ async def _methGetPropArrayCount(self, propname, valu=undef):
     @stormfunc(readonly=True)
     async def _methGetEdges(self, verb=None):
         verb = await toprim(verb)
-        todo = s_common.todo('getEdges', verb=verb)
-        async for edge in self.viewDynIter(todo, ('view', 'read')):
-            yield edge
+
+        viewiden = self.valu.get('iden')
+        self.runt.confirm(('view', 'read'), gateiden=viewiden)
```

Comment:
Is there a case where someone has lifted a node from a runtime and doesn't have view.read already?